home_test.go

```diff
@@ -32,6 +33,11 @@ func (s *fakeHomeSuite) SetUpTest(c *gc.C) {
 	utils.SetHome("/tmp/tests")
 }
 
+func (s *fakeHomeSuite) TearDownSuite(c *gc.C) {
+	s.fakeHomeSuite.TearDownSuite(c)
+	s.IsolationSuite.TearDownSuite(c)
+}
+
 func (s *fakeHomeSuite) TestHomeCreated(c *gc.C) {
 	// A fake home is created and set.
 	s.fakeHomeSuite.SetUpTest(c)
```

Comment:
We only create the fakeHomeSuite one time in SetUpSuite() thus we should call fakeHomeSuite.SetUpSuite() there, and then tear it down during TearDownSuite(), IMO.

The tests themselves are just calling SetUpTest() and TearDownTest()